react-hot-button/hot-button/src/hotButton.tsx

```diff
@@ -0,0 +1,37 @@
+import { useState } from 'react';
+import './hotButton.css';
+type Props = {
+  label: string;
+  temp?: string;
```

Comment:
I don't understand why `temp` is being passed as a prop? You are changing it in your if/else statements, so it should just be a local variable instead of a prop.